### PR TITLE
removed the isOptionKey prop

### LINKED ISSUE #446: use global alt key context
currently in our home page we are doing prop drilling to make `isOptionKey` to true, we need to use the `keyboardContext` which already exists there

### PR DESCRIPTION
**Pull request by vinayak-trivedi**
### What is the change?

closes #446 

Removed the `isOptionkey` prop, now we will use `isOptionKeyPressed` from `keyboardContext`

### Is it bug?
no

### \*Dev Tested?

- [X] Yes
- [ ] No

#### Platforms

- [X] Web

#### Browsers

- [X] Chrome
- [ ] Safari
- [ ] Firefox

### CODE REVIEW

**Comment by YashJain24-chief** [comment]
Looks good to me!

**Review by priyanshk20 — APPROVED** [review]
LGTM

**Review by YashJain24-chief — APPROVED** [review]

**Review by rupal-2505 — APPROVED** [review]

File: src/components/member-card/super-user-options/container.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useKeyboardContext } from '../../../store/keyboard/context';
+import { userContext } from '../../../store/user/user-context';
```
Comment (Maheima) [inline src/components/member-card/super-user-options/container.js:3]: Please make all the imports as we have done at other places, use @

**Review by Maheima — CHANGES_REQUESTED** [review]

File: src/components/member-card/super-user-options/container.js (hunk 1 above)
Comment (vinayak-trivedi) [reply src/components/member-card/super-user-options/container.js:3]: mentioned the reason [here](https://github.com/Real-Dev-Squad/website-members/pull/447#discussion_r992507320)

File: src/store/keyboard/context.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { ALT_KEY } from '@constants/AppConstants';
 import { createContext, useContext, useState } from 'react';
+import { ALT_KEY } from '../../constants/AppConstants';
```
[the remark this answers is not in the document]
Comment (vinayak-trivedi) [reply src/store/keyboard/context.js]: figured out a way to solve it, it is fixed in the latest commit now

File: src/components/member-card/super-user-options/container.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useKeyboardContext } from '@store/keyboard/context';
+import { userContext } from '@store/user/user-context';
+import SuperUserOptions from '.';
```
[the remark this answers is not in the document]
Comment (vinayak-trivedi) [reply src/components/member-card/super-user-options/container.js:4]: we can import it like this, as it is the index file of the same folder

[the remark this answers is not in the document]
Comment (Maheima) [reply src/components/member-card/super-user-options/container.js:4]: Got it thanks!

**Review by Maheima — CHANGES_REQUESTED** [review]
LGTM!

**Review by Maheima — APPROVED** [review]

**Pull request merged**